examples/denoise_covar2d.py

```diff
@@ -126,17 +132,17 @@
 # estimates. For the covariance estimation, the additional information of
 # the estimated mean and the variance of the noise are needed. Again, the
 # covariance matrix estimate is provided in block diagonal form.
-cov2dctf = Cov2DCTF(sim, ffbbasis)
-mean_coeff_est = cov2dctf.get_mean_ctf(coeff_noise, h_ctf_fb, h_idx)
-covar_coeff_est = cov2dctf.get_covar_ctf(coeff_noise, h_ctf_fb, h_idx, mean_coeff_est, noise_var=noise_var)
+mean_coeff_est = cov2d.get_mean_ctf(coeff_noise, h_ctf_fb, h_idx)
+covar_coeff_est = cov2d.get_covar_ctf(coeff_noise, h_ctf_fb, h_idx, mean_coeff_est, noise_var=noise_var)
 
 # Estimate the Fourier-Bessel coefficients of the underlying images using a
 # Wiener filter. This Wiener filter is calculated from the estimated mean,
 # covariance, and the variance of the noise. The resulting estimator has
 # the lowest expected mean square error out of all linear estimators.
-coeff_est = cov2dctf.get_cwf_coeffs(coeff_noise, h_ctf_fb, h_idx,
-                                    mean_coeff=mean_coeff_est,
-                                    covar_coeff=covar_coeff_est, noise_var=noise_var)
+logger.info('Get the CWF coefficients of noising images.')
+coeff_est = cov2d.get_cwf_coeffs(coeff_noise, h_ctf_fb, h_idx,
```

Comment:
This shouldn't be a function of the `RotCov2D` object, since it has nothing to do with estimating the 2D covariance. Maybe it would be better if this were a standalone function?

Reply:
You are right. Maybe we can put it in the new `Denoise` or `Coefficient` class.

Reply:
It should be in the denoiser, I think.